modules/opensearch/opensearch.module

```diff
@@ -22,6 +25,135 @@ function opensearch_ding_provider() {
 }
 
 /**
+ * Form builder; Configure Opensearch settings for this site.
+ *
+ * @ingroup forms
+ *
+ * @see system_settings_form()
+ */
+function opensearch_admin_settings($form_state) {
+  $form = array();
+
+  $form['opensearch'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Opensearch service settings'),
+    '#tree' => FALSE,
+  );
+
+  // TODO BBS-SAL: The next couple of settings seems opensearch specific, move
```

Comment:
This comment seems outdated now :)

Reply:
:+1: I have updated the comment to be focused on ting_agency.